bool reception
bool from_robot
string robot_name
float64 obj_x
float64 obj_y
